README.md

```diff
@@ -68,6 +70,27 @@ yarn install # only needed once
 yarn watch
 ```
 
+### MacOS
+
+On Mac OS we were unable to build the extension using the default clang installation.
+As a workaround we installed llvm using brew and used it to build the web extension.
+
+Install llvm
+
+```bash
+brew install llvm
+```
+
+Run the extension
+
+```bash
+cd extension;
+CC=/usr/local/opt/llvm/bin/clang AR=/usr/local/opt/llvm/bin/llvm-ar yarn install
```

Comment:
yarn install should not be concerned with these envvars

Reply:
are you suggesting that the setting of env vars be done a different line?

Reply:
I am saying, that only the `yarn watch` is concerned with the envvars `CC` and `AR`, because `yarn watch` triggers the wasm compilation. `yarn install` only installs nodejs packages and is not concerned with wasm.